docs/CONTRIBUTING.md

```diff
@@ -33,6 +33,10 @@ Netlify will try to reach the plugin author(s), and if that fails, we will add a
 
 ## Releasing
 
+Plugins are available to users as soon as the build of the Netlify site hosting `site/plugins.json` has completed. That build is triggered automatically each time a pull request is merged.
```

Comment:
What do you think about adding a time estimate, too? Like:

```suggestion
Plugins are available to users as soon as the build of the Netlify site hosting `site/plugins.json` has completed. That build is triggered automatically each time a pull request is merged, and typically takes less than a minute.
```

Reply:
I've heard Netlify is pretty fast. :rocket: